app/javascript/app/components/charts/stacked-area/stacked-area-component.jsx

```diff
@@ -50,7 +50,8 @@ class ChartStackedArea extends PureComponent {
       x: ['dataMin', 'dataMax'],
       y: ['dataMin', 'dataMax']
     };
-    if (points.length > 0) {
+    // TODO: remove hack that disables quantifications
```

Comment:
sorry for doing this, José =) But this bit wasn't approved, and I couldn't see an easier way. 🙏